testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/SocketsAndInterfacesTestCase.java

```diff
@@ -129,9 +154,7 @@ public void testAddUpdateRemove() throws Exception {
 
         logger.info("Restarting server.");
 
-        // reload server
-        op = createOpNode(null, "reload");
-        result = executeOperation(op);
+        ServerReload.executeReloadAndWaitForCompletion(getModelControllerClient());
```

Comment:
The socket-binding will be bound at this point, so this reload will be necessary. If it wasn't the assert at L128 of the current code in master would fail.